docs/install/custom-server.rst

```diff
@@ -31,11 +31,25 @@ Step 1: Installing The Littlest JupyterHub
 #. Using a terminal program, SSH into your server. This should give you a prompt where you can
    type commands.
 
+#. If your server is behind a firewall and needs a proxy:
```

Comment:
Can we move this to the 'troubleshooting section' under 'your own server' (like we have for GCP and AWS)? We can link to that more prominently here as well. I'd rather we do that than put this in here explicitly.

Reply:
I think this can be a section called 'running behind a corporate proxy' or some such, to prevent confusion with running behind a reverse proxy like nginx or apache.

Reply:
I think this must be here. The install simply does not work if it is missing. I will update text per your comment on a firewall type proxy. I can say "corporate", but maybe it is just an "IT" firewall between your server and the internet - even if it is not corporate. LMK on that.